test_files/declare_anonymous_type/declare_anonymous_type.d.ts

```diff
@@ -0,0 +1,4 @@
+// `ambientFn` is ambient, but has no `declare` keyword (which is legal TS). This test makes sure
```

Comment:
It is ambient because it lives in a '.d.ts' file. It wasn't clear to me at first, add that to the comment.

Reply:
I tried the same thing with a non-module, but it is a TS error:
```TS1046: A 'declare' modifier is required for a top level declaration in a .d.ts file.```

Reply:
Yeah, odd isn't it? I think that's a usability bug, should be consistent.